src/storage/mutate/UpdateEdgeProcessor.h

```diff
@@ -27,6 +27,12 @@ class UpdateEdgeProcessor
     void process(const cpp2::UpdateEdgeRequest& req);
 
 private:
+    enum class FilterResult {
```

Comment:
Maybe we could use a better name, for example:
```
    enum class FilterResult {
        SUCCEEDED               = 0,  // pass filter
        FILTER_OUT  = -1,  // filter out
        ERROR           = -2,  // exception when filter
    };
```

Reply:
Well, I had change this to fit our exist flavor.

Reply:
Move the structure into CommonUtils.h